Classes/Settings/SettingsViewController.swift

```diff
@@ -101,6 +102,8 @@ NewIssueTableViewControllerDelegate {
             onAccounts()
         } else if cell === githubStatusCell {
             onGitHubStatus()
+        }else if cell === reivewOnAppStoreCell {
```

Comment:
Nit: space before `else`.

Reply:
Got it. @BasThomas Whats Nit stand for?

Reply:
It’s defined [here](https://www.merriam-webster.com/dictionary/nit) as a “minor shortcoming”. In this case that it doesn’t really need fixing, but it would be nice.